### PR TITLE
Added Tamil Data

### PR DESCRIPTION
**Pull request by anupraman2000**

### CODE REVIEW

File: locales/ta/data.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    doctorTele: `Doctor/Telemedicine/Consultation`,
-    doctorDes: `Specialisation, City, Contact`,
-    supportGroup: `Support Groups (WhatsApp, Telegram etc)`,
-    supportDes: `Platform, Category, Contact`,
-    applyVol: `Apply as volunteer`,
-    applyDes: `Skills, Contact`,
-    govtContact: `Government Contact`,
-    govtDes: `Source, City, Contact`,
-    fillTheForm: `Fill the Form`
+    கேள்வி: `உங்களிடம் சரிபார்க்கப்பட்ட தரவு இருக்கிறதா மற்றும் தரவுத்தளத்தில் சேர்க்க விரும்புகிறீர்களா?`,    sourceType: `Source, Type, Delivery`,
```
Comment (vandanabhandari) [inline locales/ta/data.js:2]: @anupraman2000 you should not translate the key

**Review by vpremk — CHANGES_REQUESTED** [review]
Plz remove the key translations

**Review by Deep-Codes — CHANGES_REQUESTED** [review]
Your Changing the Keys don't change them.

File: locales/ta/data.js (hunk 1 above)
Comment (Deep-Codes) [reply locales/ta/data.js:2]: Gotta update the prev PR

Comment (vandanabhandari) [reply locales/ta/data.js:2]: prev one seemed ok. I checked
https://github.com/coronasafe/life/pull/123/files

Comment (Deep-Codes) [reply locales/ta/data.js:2]: Because I made the changes , there was 1 key translated,

Comment (vandanabhandari) [reply locales/ta/data.js:2]: oh got it.. will check the next one to make sure its done right

**Comment by vandanabhandari** [comment]
Closing this as @anupraman2000 created https://github.com/coronasafe/life/pull/124

**Pull request closed without merging**